test/cat.jl

```diff
@@ -62,6 +62,15 @@ module TestCat
         @test df[2] == collect(1:10)
     end
 
+    @testset "hcat ::Vectors" begin
```

Comment:
Either merge with previous block, or find a different name?

Reply:
There were earlier such duplicate blocks that is why I left it. Now I merge them all.